oct/ansible/oct/roles/remote-sync/tasks/main.yml

```diff
@@ -27,7 +27,7 @@
   when: origin_ci_sync_remote not in origin_ci_sync_remotes_probe.stdout and origin_ci_sync_address is defined
 
 - name: inspect the remote for the address if we do not have an address
-  shell: '/usr/bin/git remote show {{ origin_ci_sync_remote }} | grep "Fetch URL" | cut -c 14- '
+  command: '/usr/bin/git remote get-url {{ origin_ci_sync_remote }}'
```

Comment:
Do we want to remove absolute path from the command? If yes, we could try to use `environment` (although I'm not 100% sure that it will work with `command`): http://docs.ansible.com/ansible/faq.html#how-can-i-set-the-path-or-any-other-environment-variable-for-a-task-or-entire-playbook